### PR TITLE
Alex infra deploy

### PR DESCRIPTION
**Pull request by agiannini**
Cloudformation scripts to provision AWS env

### CODE REVIEW

File: cfn/deploy_ecs.yaml [hunk 1]
```diff
@@ -0,0 +1,329 @@
...
+    Properties:
+      EphemeralStorage:
+        SizeInGiB: 40
+      TaskRoleArn: !Ref DjangoTaskExecutionRole
+      ContainerDefinitions:
+      - Command:
+        - us-east-1.compute.internal
+        - dle.local
+        Essential: false
+        Image: docker/ecs-searchdomain-sidecar:1.0
```
Comment (ColeDCrawford) [inline cfn/deploy_ecs.yaml:185]: This is the built image we push to ECR? Just curious, a bit confused by the name

File: cfn/deploy_ecs.yaml [hunk 2]
```diff
@@ -0,0 +1,329 @@
...
+          ContainerName: Django_ResolvConf_InitContainer
+        Environment:
+        - Name: AWS_ACCESS_KEY
+          Value: !Ref awsAccessKey
+        - Name: AWS_ACCESS_KEY_ID
+          Value: !Ref awsAccessKey
+        - Name: AWS_SECRET_ACCESS_KEY
+          Value: !Ref awsSecretKey
+        - Name: ALLOWED_HOSTS
+          Value: '*'
```
Comment (ColeDCrawford) [inline cfn/deploy_ecs.yaml:206]: This should get updated right?

File: cfn/deploy_ecs.yaml [hunk 3]
```diff
@@ -0,0 +1,329 @@
...
+        - Name: POSTGRES_PASSWORD
+          Value: !Ref rdsPword 
+        - Name: POSTGRES_PORT
+          Value: "5432"
+        - Name: POSTGRES_USER
+          Value: postgres
+        - Name: SECRET_KEY
+          Value: !Ref secretPword
+        - Name: STACK_VERSION
+          Value: 8.5.0
```
Comment (ColeDCrawford) [inline cfn/deploy_ecs.yaml:252]: This should be 8.7.0

File: cfn/deploy_ecs.yaml [hunk 4]
```diff
@@ -0,0 +1,329 @@
...
+        Name: django
+        PortMappings:
+        - ContainerPort: 8000
+          HostPort: 8000
+          Protocol: tcp
+      Cpu: "4096"
+      ExecutionRoleArn:
+        Ref: DjangoTaskExecutionRole
+      Family: dle-django
+      Memory: "8192"
```
Comment (ColeDCrawford) [inline cfn/deploy_ecs.yaml:273]: These the right stats?

**Review by ColeDCrawford — APPROVED** [review]
Couple questions about how specific references but otherwise LGTM.

Where can we configure the Kibana / Elastic MEM_LIMIT? We should tweak that based on the EC2 size. 1GB Kibana, ~1GB for the host, the rest for Elastic. That means we can't run ad-hoc Django tasks on the EC2 any more but we need to give Elastic more oomph as it seems to be running slow, probably because the indices have grown and we're not holding everything in RAM anymore.

File: cfn/deploy_ecs.yaml (hunk 1 above)
Comment (agiannini) [reply cfn/deploy_ecs.yaml:204]: no, the image is defined in the SSM param. this is for the init container

File: cfn/deploy_ecs.yaml (hunk 3 above)
Comment (agiannini) [reply cfn/deploy_ecs.yaml]: updated

File: cfn/deploy_ecs.yaml (hunk 4 above)
Comment (agiannini) [reply cfn/deploy_ecs.yaml]: lowed to 2 and 4, created param.

**Pull request merged**